### DISCUSSION TITLE
Possible to have a higher resolution version of this card ?

### DISCUSSION THREAD

**Discussion by Tipalo** [tomasrudh/analogclock#26] [Ideas]
This is a great card and having more options would be very nice.  A photo-realistic version would be very nice for wall mounted displays, tablets and Google Hubs. Some bling for the front end of the home.

Higher resolution with skin-able faces would be very cool.

Thanks
D

**Comment by tomasrudh** [comment]
Thanks for the suggestion, I don't have plans to implement right now, maybe if there are more requests for it.
